modules/test/playwright/helpers/CommerceMiniumApiHelper.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {ApiHelpers, DataApiHelpers} from './ApiHelpers';
+
+export class CommerceMiniumApiHelper {
+	readonly apiHelpers: ApiHelpers | DataApiHelpers;
+	readonly dataApiHelpers: DataApiHelpers;
+
+	constructor(apiHelpers: ApiHelpers | DataApiHelpers) {
+		this.apiHelpers = apiHelpers;
+	}
+
+	async loadCleanUpData(site: Site) {
```

Comment:
Use the utility done here: https://github.com/liferay-commerce/liferay-portal/pull/4994/commits/63db7c5f40383f785c331923be43682541ea2f0a